packages/search/lib/Documents.js

```diff
@@ -160,12 +161,12 @@ const getElasticDoc = (
     __sort: {
       date: meta.publishDate
     },
-    resolved,
     id,
     commitId,
     versionName,
     milestoneCommitId,
     meta,
+    resolved: !_.isEmpty(resolved) ? resolved : undefined,
```

Comment:
Why needed? Because you now also index empty objects in the new logic? Maybe just prevent indexing empty objects in the first place.

Reply:
I do not understand this comment just yet.